### PR TITLE
Added some PROV-JSON files and scripts

### PR DESCRIPTION
**Pull request by alison-li**
Sources:
* https://github.com/RobertMyles/Various_R-Scripts/tree/master/R
* https://github.com/chrisalbon/code_r

There's not as many in the first repo but the second one contains a lot. I picked a few out but we can peruse for more later and take out some if we don't like any committed here for now.

### CODE REVIEW

File: test/scripts/roll_call_matrix.R [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    
+    miss <- which(is.na(rollCallMatrix))
+    StanObject <- rollCallMatrix[-miss]
+    Legis_data <- Leg
+    list_rc_stan <- list(StanObject = StanObject, 
+                         Legis_data = Legis_data)
+    message("Returning list of rollcall vector and legislator \ncharacteristics for Stan")
+    return(list_rc_stan)
+    
+  } else if(pscl == FALSE & Stan == FALSE & Legis == FALSE){
```
Comment (jyoo980) [inline test/scripts/roll_call_matrix.R:100]: I'm the biggest @Alison-Li stan after this PR

Comment (alison-li) [reply test/scripts/roll_call_matrix.R:100]: LOL I didn't even notice

**Review by jyoo980 — CHANGES_REQUESTED** [review]
Can you remove `test/.DS_Store`? Adding an update to the `.gitignore` to ignore all `.DS_Store` files might be a good idea

**Comment by jyoo980** [comment]
Adding 
```
/test
```
to the `.gitignore` would exclude the entire folder from being committed + pushed (is that what you meant to do?). I made a minor modification that would make it only ignore `.DS_Store` files.

**Review by jyoo980 — APPROVED** [review]

**Pull request merged**